bridge_ui/src/index.js

```diff
@@ -15,9 +15,14 @@ import { TerraWalletProvider } from "./contexts/TerraWalletContext.tsx";
 import ErrorBoundary from "./ErrorBoundary";
 import { theme } from "./muiTheme";
 import { store } from "./store";
-import { AlephiumWalletProvider } from "@alephium/web3-react";
+import { AlephiumWalletProvider, createWalletConnectConnector, createDesktopWalletConnector } from "@alephium/web3-react";
 import { CLUSTER, ALEPHIUM_BRIDGE_GROUP_INDEX } from "./utils/consts";
 
+const connectors = {
+  walletConnect: createWalletConnectConnector({customStoragePrefix: 'alephium'}),
+  desktopWallet: createDesktopWalletConnector({customStoragePrefix: 'alephium'})
```

Comment:
Maybe we should enable this by default in our walletconnect provider? There could be Other projects need to use multiple connectors.

Reply:
Sounds good, will update